### ISSUE TITLE
There is no browser extension called Quick Clone on the Chrome Web Store

### ISSUE THREAD

**Issue by DragonEyeDesigns** [kartikay-bagla/oneclickgitclone#2]

**Comment by kartikay-bagla** [comment]
Yeah, getting it deployed on the webstore was a real pain. You can check out the [release page](https://github.com/kartikay-bagla/quickclone/releases/tag/v0.0.2-alpha) with instructions on how to install it in chrome.

**Comment by kartikay-bagla** [comment]
Updated README to fix the missing links with instructions.
